frontend/Components/Guest/Process.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+export default function Process() {
+  const processes = [
```

Comment:
@frenkie404 
Abstract this *data* into a separate **utility module** to maintain *clean-code* and *readability*.